ignite/metrics/gan/utils.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, return_features: bool, device: Union[str, torch.device] = "cp
             raise RuntimeError("This module requires torchvision to be installed.")
         super(InceptionModel, self).__init__()
         self._device = device
-        self.model = models.inception_v3(pretrained=True).to(self._device)
+        self.model = models.inception_v3(weights=models.Inceaaaption_V3_Weights.DEFAULT).to(self._device)
```

Comment:
Here is a typo in `Inceaaaption_V3_Weights`. Another point here is that this will fail with old pytorch version and min supported version is 1.3.1. See https://github.com/pytorch/ignite/actions/runs/2864338520

Reply:
Sorry for the typo. Let me correct this.
For minimum pytorch version,  may I ask what is typical methods to deal with this.
Is this ok with condition like this?

```
if torch.__version__ < 1.3.1:
    ~~~~
```

Reply:
Here is the way we check for torch version:
https://github.com/pytorch/ignite/blob/3d9ba50bba42438a63183b3db0d4cc3a5e9b01ac/ignite/metrics/gan/fid.py#L197

And to clarify, our code needs to support pytorch >= 1.3.1, for below we do not care anymore.